x-pack/test/functional_with_es_ssl/config.ts

```diff
@@ -50,8 +50,8 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
     pageObjects,
     // list paths to the files that contain your plugins tests
     testFiles: [
-      resolve(__dirname, './apps/discover'),
       resolve(__dirname, './apps/triggers_actions_ui'),
+      resolve(__dirname, './apps/discover'),
```

Comment:
Needed to change the order of the tests here. If the Discover test runs first, it will clean up its data & indices. Before my change, in case of no data, Discover would redirect to `kibana_overview` app. With this change, in case of no data, it stays within Discover. So the `triggers_actions_ui` test starts from Discover, and the user for the test doesn't have Discover permissions, so this was throwing an error. 
Changing the order of the test seemed like a cleaner solution than adding Discover-specific privileges to the user, since they are not really needed.